src/lineAnalysis.js

```diff
@@ -0,0 +1,76 @@
+import * as cellValue from '../src/cellValue';
+import * as gameHelper from '../src/components/game/gameHelper';
+
+export function isThereAWinner(grid) {
```

Comment:
`isThereAWinner` implies to me that a boolean is going to be returned. Maybe a different name for this?